sdk/resourcemanager/azure-resourcemanager-eventhubs/src/main/java/com/azure/resourcemanager/eventhubs/implementation/EventHubImpl.java

```diff
@@ -0,0 +1,532 @@
+            this.ensureSettings().withIntervalInSeconds(sizeInSeconds);
+            return this;
+        }
+
+        public CaptureSettings withDataCaptureWindowSizeInMB(int sizeInMB) {
+            this.ensureSettings().withSizeLimitInBytes(sizeInMB * 1024 * 1024);
+            return this;
+        }
+
+        public CaptureSettings withDataCaptureFileNameFormat(String format) {
+            this.ensureSettings().destination().withArchiveNameFormat(format);
+            return this;
+        }
+
+        public CaptureDescription validateAndGetSettings() {
+            if (this.newSettings == null) {
+                return this.currentSettings;
+            } else if (this.newSettings.destination() == null
+                    || this.newSettings.destination().storageAccountResourceId() == null
+                    || this.newSettings.destination().blobContainer() == null) {
+                throw logger.logExceptionAsError(new IllegalStateException(
+                    "Setting any of the capture properties requires "
+                        + "capture destination [StorageAccount, DataLake] to be specified"));
+            }
+            if (this.newSettings.destination().name() == null) {
+                this.newSettings.destination().withName("EventHubArchive.AzureBlockBlob");
+            }
+            if (this.newSettings.encoding() == null) {
+                this.newSettings.withEncoding(EncodingCaptureDescription.AVRO);
+            }
+            return this.newSettings;
+        }
+
+        private CaptureDescription ensureSettings() {
+            if (this.newSettings != null) {
+                return this.newSettings;
+            } else if (this.currentSettings == null) {
+                this.newSettings = new CaptureDescription().withDestination(new Destination());
+                return this.newSettings;
+            } else {
+                // Clone the current settings to new settings (one time)
+                //
+                this.newSettings = cloneCurrentSettings();
+                return this.newSettings;
+            }
+        }
+
+        private Mono<Void> createContainerIfNotExistsAsync(final StorageAccount storageAccount,
+                                                                    final String containerName) {
+            return getBlobClientAsync(storageAccount)
+                .flatMap(blobServiceAsyncClient -> blobServiceAsyncClient
+                    .getBlobContainerAsyncClient(containerName)
+                    .exists()
+                    .flatMap(aBoolean -> {
+                        if (aBoolean) {
+                            return Mono.empty();
+                        }
+                        return blobServiceAsyncClient.getBlobContainerAsyncClient(containerName).create();
+                    }));
+        }
+
+        private Mono<BlobServiceAsyncClient> getBlobClientAsync(final StorageAccount storageAccount) {
```

Comment:
Here we adopt `com.microsoft.azure: azure-storage` to `com.azure: azure-storage-blob`, comparing to [previous code](https://github.com/Azure/azure-libraries-for-java/blob/master/azure-mgmt-eventhub/src/main/java/com/microsoft/azure/management/eventhub/implementation/EventHubImpl.java#L558-L602).

Reply:
Remove this as we can do in mgmt storage.